hivemind/modules/api/models/article.py

```diff
@@ -0,0 +1,28 @@
+"""
+Article Model for API Module.
+"""
+from django.db import models
+
+from hivemind.modules.api.models.base_model import BaseModel
+
+
+class Article(BaseModel):
+    """
+    This is the Article Model.
+    """
+    headline = models.CharField(
+        'Headline',
+        max_length=255,
+        help_text='The headline of the article.',
+    )
+    content = models.TextField(
+        'Content',
+        help_text='The contents of the article.',
+    )
+
+    # pylint: disable=too-few-public-methods
+    class Meta:
```

Comment:
Please add a `__str__` method for this class. Should just ideally display the `headline`